### PR TITLE
Adding new file

### PR DESCRIPTION
**Pull request by Jucerweb**
Adding new information to test git

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-testing changes
+- Testing changes
```
Comment (JuanSaieh) [inline README.md:1]: why did you change this line?

**Review by JuanSaieh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jucerweb) [reply README.md:1]: Because reasons!
